pkgs/top-level/python-packages.nix

```diff
@@ -10597,7 +10572,7 @@ in {
       sha256 = "0wxvpmjdzk0aajk33y4himn3wqjx7k0aqlka9j8ay3yfav78bdq0";
     };
 
-    propagatedBuildInputs = with self; [ django_1_7 ];
+    propagatedBuildInputs = with self; [ django_1_8 ];
```

Comment:
The reason being we don't want risking multiple versions of a package in the same closure.